app/labor/rate_limit_checker.rb

```diff
@@ -9,22 +9,13 @@ def initialize(user = nil)
     @user = user
   end
 
-  class UploadRateLimitReached < StandardError; end
-  class DailyFollowAccountLimitReached < StandardError; end
+  UploadRateLimitReached = Class.new(StandardError)
+  DailyFollowAccountLimitReached = Class.new(StandardError)
 
   def limit_by_action(action)
-    result = case action
```

Comment:
I'm not a big fan of `case` statements like this, so went for an approach with `public_send` and dedicated methods instead. This also means that instead of 

```ruby
RateLimitChecker.new.limit_by_action("comment_creation")
```

we can also do 

```ruby
RateLimitChecker.new.check_comment_creation_limit
```
which is a bit easier for discoverability/code navigation.

Maybe we can even leave off the `_limit` postfix, it seems a bit redundant.